### ISSUE TITLE
Need to add park trees data (longitude and latitude) to kde

### ISSUE THREAD

**Issue by jandarli** [janicedarling/CSC59969Project#2]

**Comment by JetlirLajqi** [comment]
Ok I will have park data by 7pm, do you need x and y corrdinate of every
tree??

**Comment by jandarli** [comment]
Yeah look at script.js and see how I'm creating the heat map for the tree data. You wouldn't be able to get the coordinates for every tree. My suggestion was to find out how you could do a geofence of the parks for example central park and get a sample of coordinates in that geofence.

**Comment by JetlirLajqi** [comment]
Ok, Im grabbing general park perimeters from each borough and grabbing
coordinates of major tree areas

**Comment by JetlirLajqi** [comment]
Ill commit the stats soon

**Comment by jandarli** [comment]
Put them on the python-scripts branch or create a branch specifically for estimating the trees before pushing to the master
